action.yaml

```diff
@@ -176,6 +176,14 @@ runs:
             run: symfony server:ca:install
             shell: bash
 
+        -   name: Remove default Google Chrome installation
+            run: |
+                which google-chrome && google-chrome --version || echo "Google Chrome successfully removed"
+                sudo dpkg --remove google-chrome-stable
+                sudo apt-get purge -y google-chrome-stable
+                which google-chrome && google-chrome --version || echo "Google Chrome successfully removed"
+            shell: bash
```

Comment:
Should be moved to the top of the `Prepare for end-to-end tests` section and made conditional based on the `e2e_js` setting, no point in reinstalling chrome, when we're not using it 🤷